### DISCUSSION TITLE
Wear Os App/Tile

### DISCUSSION THREAD

**Discussion by djpalmis** [teslamate-org/teslamate#1947] [Ideas]
It would be nice to integrate teslamate with a wearable app such as a Tile for Wear Os.

I check many times the website just to check the state, SoC, temperature and 100% range estimation. 

Those information would perfectly fit in a tiny smartwatch screen!

I don't know if MQTT would be the right way to send those informations.

**Comment by DrMichael** [comment]
I played around a bit with nodered getting MQTT messages and send them out via email or web call. Not so hard to set up...

**Reply by djpalmis** [reply]
Unfortunately I'm so unskilled about this matters. I hope some developer would be so nice to release an app like that.
